common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/InvisibilityReason.scala

```diff
@@ -8,4 +8,5 @@ object InvisibilityReason {
   case class CopyrightNotCleared(info: String) extends InvisibilityReason
   case class SourceFieldMissing(info: String) extends InvisibilityReason
   case class InvalidValueInSourceField(info: String) extends InvisibilityReason
+  case class UnlinkedHistoricalLibraryMiro() extends InvisibilityReason
```

Comment:
Any reason this needs to be a `case class` rather than a `case object`?

Reply:
Just for consistency with the others tbh

Reply:
I don't think inconsistent, I think others are that way because have an `info` argument. Maybe we want an `info` string for this too?

Reply:
Yeah, I couldn't find anything that would be useful `info` for this - the others use it to say (eg) what their source is. I'll make it an `object`